index.html

```diff
@@ -320,41 +340,47 @@ <h3>BrowserCaptureMediaStreamTrack</h3>
                 </p>
                 <ul>
                   <li>
-                    If [=this=] track is [=uncropped=], the user agent MUST only accept [=valid
-                    CropTargets=].
+                    Let |valid| be the result of running [=determine if crop target is valid=] on
+                    |cropTarget|.
+                  </li>
+                  <li>
+                    Let |success| be a [=boolean=], initially set to false.
+                  </li>
+                  <li>
+                    If [=this=] track is [=uncropped=] and |valid| is true, set |success| to true.
                   </li>
                   <li>
-                    If [=this=] track is [=cropped=], the user agent MUST accept either [=valid
-                    CropTargets=] or {{undefined}}.
+                    If [=this=] track is [=cropped=], |cropTarget| is either {{undefined}} or |valid|
+                    is true, set |success| to true.
                   </li>
                 </ul>
                 <p>
-                  If the user agent does not accept <var>cropTarget</var>, return a {{Promise}}
-                  [=rejected=] with an {{UnknownError}}.
+                  If |success| is false, return a {{Promise}} [=rejected=] with an {{UnknownError}}.
                 </p>
               </li>
               <li>Let <var>p</var> be a new {{Promise}}.</li>
               <li>
                 <p>Run the following steps in parallel:</p>
                 <ol>
                   <li>
-                    If <var>cropTarget</var> is neither {{undefined}} nor a [=valid CropTarget=],
-                    reject <var>p</var> with a {{NotAllowedError}} and abort these steps.
+                    If <var>cropTarget</var> is neither {{undefined}} or if calling [=determine if
```

Comment:
Is this now redundant with the steps around `|success|`?
(And nit: neither-nor, not neither-or, I think.)

Reply:
Dropped this indeed. Does this mean that NotAllowedError was never something that could be triggered?

Reply:
It can never trigger because _cropTarget_ cannot be `undefined`. It can be `null` because it is nullable. `NotAllowedError` implies permission failure and is the wrong error here. Maybe `InvalidStateError`.

There are lots of problems with this spec, the biggest being it's shape lacks consensus, and we've been attacking them from big to small. FPWD is meant to be the beginning of standardization, not the end.

Reply:
I understand the current PR as making an iterative improvement that does **not** introduce any new non-consensus issues. If any non-consensus issues are mistakenly introduced by this PR, @jan-ivar, please flag them and we can remove them from the PR.